packages/fhir-group-management/src/components/CommodityAddEdit/Form.tsx

```diff
@@ -97,11 +97,11 @@ const CommodityForm = (props: GroupFormProps) => {
       }}
       initialValues={initialValues}
     >
-      <FormItem id="id" name={id} label={t('Commodity Id')}>
+      <FormItem hidden={true} id="id" name={id} label={t('Commodity Id')}>
         <Input placeholder={t('(Auto generated)')} disabled={true} />
```

Comment:
This does not look like it gets programatically filled, i.e `value=""`, is that intentional?

Reply:
The value is implicitly bound to the input, this happens because of the name attribute in FormItem